server/src/test/java/org/elasticsearch/common/RoundingTests.java

```diff
@@ -195,6 +195,16 @@ public void testTimeUnitRoundingDST() {
         assertThat(tzRounding_chg.round(time("2014-11-02T06:01:01", chg)), isDate(time("2014-11-02T06:00:00", chg), chg));
     }
 
```

Comment:
Hmm, do you know if we have any serialization tests for `Rounding`?  I was looking to see if we test that the `id()` bytes are correct and don't accidentally change (similar to what we do with `AbstractWriteableEnumTestCase` enum tests)... but couldn't find any serialization tests at all.

If we have them somewhere, let's add a test for the id byte.  If not, probably too much to add to this PR but we should file a ticket so we don't forget to add some tests... makes me uneasy that such a widespread class doesn't have serialization tests :)

Reply:
I believe we end up testing serialization as part of testing things like extended bounds bucket response. Adding a unit test for just this class's serialization makes sense to me though. I figured I'd wait until I used it in a context where we serialized it but since I'm writing the serialization code now I probably ought to write the test now.

Reply:
I pushed a wire test case.